styles.css

```diff
@@ -24,12 +24,12 @@ footer {
     background-color: lightgreen;
 }
 /*For desktop screens*/
-@media screen and (min-width: 1025px) {
```

Comment:
Ik zie dat je 2 keer een breakpoint / @media hebt toegepast (regel 27 en regel 47). Je zou dit met 1 x een breakpoint / @ media kunnen doen. Begin de CSS style op te maken voor het kleinste scherm (in dit geval voor de mobiele telefoon) en voeg onder aan de code de breakpoint met @media toe  welke dan geld voor de desktop.  Functioneel gezien werkt het wel gewoon goed zoals ik het heb kunnen constateren